content/administration/upgrade.rst

```diff
@@ -232,7 +232,7 @@ The upgrade of:
 
 * standard applications
 * Studio customization (as long as the :guilabel:`Studio` app is still active)
-* customizations done by our consulting and developer services *if* they are covered by a
+* customizations and developer services *if* they are covered by a
   ‘Maintenance of Customisations’ subscription
```

Comment:
IINM, the `developer services` reffers also to who did the customizations
IMO, we should remove all the `done by our consulting and developer services` text and just leave:

```suggestion
* customizations *if* they are covered by a ‘Maintenance of Customisations’ subscription
```